content-webapp/src/main/webapp/js/newsUtils.js

```diff
@@ -30,4 +30,16 @@ export function convertDate(date) {
 
 export function pad(n) {
   return n < 10 && `0${n}` || n;
-}
\ No newline at end of file
+}
+
+export const NewsUpdateType = {
```

Comment:
```suggestion
export const newsUpdateType = {
```
I think that you can declare that inti newsConstants.js instead of newsUtils.js

Reply:
done!